setup.py

```diff
@@ -7,7 +7,9 @@
 install_requires = [
     'PyYAML>=3.0',
     'future',
-    'pyproj>=2'
+    'pyproj>=2',
+    'jsonschema',
```

Comment:
Do we need documentation for this, as e.g. seen here: https://mapproxy.github.io/mapproxy/latest/install.html ?

Reply:
We could document every dependency there ... I don't know if that is really necessary?